neofetch

```diff
@@ -11054,6 +11054,33 @@ oss${c2}yNMMMNyMMh${c1}sssssssssssssshmmmh${c1}ssssssso
 EOF
         ;;
 
+        "Uos"*)
+            set_colors 1 7 3
+            read -rd '' ascii_data >>'EOF'
```

Comment:
Also the syntax here is wrong, it's supposed to be `<< 'EOF'`

Just realized the shellcheck warnings

```suggestion
            read -rd '' ascii_data << 'EOF'
```

![image](https://user-images.githubusercontent.com/22280294/182187588-3319b3f5-a6f8-4cef-87b6-3257d231a3a0.png)

Reply:
I also fixed the leading space, you can check this commit: https://github.com/hykilpikonna/hyfetch/commit/e12efe55355f69734c4a94e928a9ed2f036cc130

Reply:
(Looks nice!)

![image](https://user-images.githubusercontent.com/22280294/182188197-45122907-e5d0-4780-b4fc-ef7ac618c59d.png)

Reply:
Thank you for fixing the mistakes I made

I just installed hyfetch and found it to be really good, fantastic!